### PR TITLE
test: add run_k8s coverage

### PR DESCRIPTION
**Pull request by IyadhKhalfallah**
---

- [ ] This PR references an open issue
- [ ] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

**Review by azayz — APPROVED** [review]

**Review by azayz — COMMENTED** [review]
Can we try to get the coverage on run k8s even higher, it'S 72% now but I think we can get more

**Review by joschkabraun — CHANGES_REQUESTED** [review]
Nice changes! Let's push the test coverage higher than it currently is: https://app.codecov.io/gh/jina-ai/now/pull/1188/blob/now/run_all_k8s.py

File: tests/unit/test_run_all_k8s.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+def test_compare_flows_no_flow_ids(mocker: MockerFixture):
+    kwargs = {
+        'path_score_calculation': 'path/to/file',
+        'dataset': 'test',
+        'limit': 1,
+        'disable_to_datauri': True,
+        'results_per_table': 20,
+    }
+    mock_json = {'test': 'test'}
+    mock_file = mocker.mock_open(read_data='test')
```
Comment (joschkabraun) [inline tests/unit/test_run_all_k8s.py:53]: not sure how to patch reading the file but the file should have a structure of e.g.
```
{
  "lottiefiles-v3-elastic-nowapi-9a817214b3": [
    [],
    [
      ["text_0", "text_0", "encodersbert", 1]
    ],
    [
      ["text_0", "video_0", "encoderclip", 1]
    ]
  ]
}
```
we need to make sure that the test only passes with such formatting

**Review by joschkabraun — CHANGES_REQUESTED** [review]

**Review by joschkabraun — APPROVED** [review]
After this, LGTM!

File: tests/unit/test_run_all_k8s.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        'results_per_table': 20,
+    }
+
+    mocker.patch(
+        'now.compare.compare_flows.compare_flows_for_queries',
+        return_value='PASS',
+    )
+    mocker.patch(
+        'now.run_all_k8s.get_docarray',
+        return_value=[MMStructure(is_multimodal=True)],
```
[the remark this answers is not in the document]
Comment (IyadhKhalfallah) [reply tests/unit/test_run_all_k8s.py:59]: Merged without paying attention to these comments. I'll add them to #1200

**Pull request merged**